website/project/model.py

```diff
@@ -2756,7 +2755,7 @@ def remove_contributor(self, contributor, auth, log=True):
         self.save()
 
         #send signal to remove this user from project subscriptions
-        auth_signals.contributor_removed.send(contributor, node=self)
+        project_signals.contributor_removed.send(contributor, node=self)
```

Comment:
This is fine for now to keep compat, but it probably makes more sense to make `self` the sender, not the contributor.